File: SNACKS/Models/MovimientoCaja.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace SNACKS.Models
{
    public class MovimientoCaja
    {
        public int IdMovimientoCaja { get; set; }
        public int IdCaja { get; set; }
        public string TipoMovimiento { get; set; }
        public int? IdCajaOrigen { get; set; }
        public int? IdPedido { get; set; }
        public int? IdIngresoInsumo { get; set; }
        public string Glosa { get; set; }
        public DateTime Fecha { get; set; }
        public decimal Importe { get; set; }
        public Usuario Usuario { get; set; }
    }
}

File: SNACKS/ClientApp/src/app/movimientos-caja/movimiento-caja.ts
import { IUsuario } from "../usuarios/usuario";

export interface IMovimientoCaja {
  idMovimientoCaja: number;
  idCaja: number;
  tipoMovimiento: string;
  idCajaOrigen: number;
  idPedido: number;
  idIngresoInsumo: number;
  glosa: number;
  fecha: Date;
  importe: number;
  usuario: IUsuario;
}
